content/docs/1.9.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

```diff
@@ -106,14 +111,17 @@ The Longhorn system backup offers the following volume backup policies:
 
 #### Create a Recurring Backup Job Using the Longhorn UI
 
-1. Go to the `Recurring Job` page.
```

Comment:
The term `screen` is unfamiliar to me when it comes to navigating the UI.

cc @jillian-maroket @derekbit @innobead